### ISSUE TITLE
Mouse movement laggy

### ISSUE THREAD

**Issue by Derpford** [janusvr/janus#42]
There's some kind of weirdness going on with the mouse in the Steam version--it seems like there's an (extremely low) upper limit to how far I can turn with the mouse, and this makes using the mouse extremely annoying. I tried using a controller, but my controller appears to just not be working right now; I will update this issue if I can get it working.

**Comment by Derpford** [comment]
Just got controller working again, and it's similarly really slow. This makes Janus nearly impossible to use in desktop mode. I'm not sure why this is an issue either--I know there are earlier versions which don't have this issue.

**Comment by y-takashina** [comment]
I have the same issue in Oculus Go. I wonder the "sea" that always appears background in every VR page harms the performance of the app while it seems completely unnecessary...

**Comment by JamesMcCrae** [comment]
Is there anything unique about your setup (e.g. multi-monitor?)

**Comment by Derpford** [comment]
Nope. Single monitor, reasonably good gfx card (AMD brand, I believe it's a 560). I *am* running through Wine/Proton right now.

**Comment by timschwartz** [comment]
I am experiencing this on Linux. I've tried the binary version from janusvr.com and compiling the latest git.

Running the Windows version in Wine works ok, until it crashes for other reasons.

**Comment by source83** [comment]
I have experienced this multiple times too, try launching janusvr with `-novsync` and see if it helps for now.
